src/infra/config/dex/okx/file.rs

```diff
@@ -13,10 +13,17 @@ use {
 #[derive(Deserialize)]
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 struct Config {
-    /// The versioned URL endpoint for the OKX swap API.
-    #[serde(default = "default_endpoint")]
+    /// The URL endpoint for the OKX swap API for sell orders (exactIn mode).
+    /// Uses V6 API by default.
+    #[serde(default = "default_sell_orders_endpoint")]
     #[serde_as(as = "serde_with::DisplayFromStr")]
-    endpoint: reqwest::Url,
+    sell_orders_endpoint: reqwest::Url,
+
+    /// The URL endpoint for the OKX swap API for buy orders (exactOut mode).
+    /// Uses V5 API. If not specified, buy orders will be declined.
```

Comment:
> Uses V5 API

This is a bit confusing because I'm guessing you mean "uses v5 api calls" but some other endpoint could be placed, it would fail but I think you see my point.


```suggestion
    /// If specified, the URL must point to the V5 API; if not specified, buy orders will be declined.
```